config/locales/en.yml

```
@@ -361,13 +361,16 @@ en:
 
   dry_validation:
     errors:
+      defined: "must not be defined."
       int?: "must be an integer."
       filled?: "must be filled."
       greater_or_equal_zero: "must be greater or equal to 0."
       not_found: "not found."
       rules:
         copy_workflow_from:
           workflow_missing: "has no own workflow."
+        custom_field:
+          format_not_supported: "format is of unsupported type."
```

:green_circle: In case this translation stays (see my remark about indicating which format is unsupported): The usage of "format" and "type" in the same sentence reads weird to my eyes.

How about simplifying it to something like

```suggestion
          format_not_supported: "format is unsupported."
```